arches/app/src/components/ControlledListManager/LetterCircle.vue

```diff
@@ -0,0 +1,56 @@
+<script setup lang="ts">
+import { computed } from "vue";
+
+import type { ControlledList, ControlledListItem, Selectable } from "@/types/ControlledListManager";
+
+const props = defineProps<{ labelled: Selectable }>();
+
+const color = computed(() => {
+    if ((props.labelled as ControlledList).search_only !== undefined) {
+        return 'midnightblue';
+    }
+    const item = props.labelled as ControlledListItem;
+    if (item.guide) {
+        return 'bisque';
+    }
+    return 'darkorchid';
+});
+
+const letter = computed(() => {
```

Comment:
Let's chat about this

Reply:
My preference would be for a non-textual icon, but I don't have any ideas at the moment. I'm hoping we can tackle this as follow-up work.